service/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/search/FilterMap.java

```diff
@@ -232,7 +230,7 @@ public void populateFilterMappings() {
         filterMappings.put(SearchFilter.VESSEL_NAME, new FilterDetails("fa.vesselTransportMeans vt", "vt.name IN (:" + VESSEL_IDENTITY_NAME + ")"));
         filterMappings.put(SearchFilter.VESSEL_IDENTIFIRE, new FilterDetails("vt.vesselIdentifiers vi", "vi.vesselIdentifierId IN (:" + VESSEL_IDENTIFIRE + ")"));
         filterMappings.put(SearchFilter.VESSEL_GUIDS, new FilterDetails("fa.vesselTransportMeans vtMeans", "vtMeans.guid IN (:" + VTM_GUIDS + ")"));
-        filterMappings.put(SearchFilter.PURPOSE, new FilterDetails(FLUX_REPORT_DOC_TABLE_ALIAS, "flux.purposeCode IN (:" + PURPOSE_CODE + ")"));
+        filterMappings.put(SearchFilter.PURPOSE, new FilterDetails(" ", "fa.fluxReportDocument_PurposeCode IN (:" + PURPOSE_CODE + ")"));
```

Comment:
The String-with-only-a-space looks funny but I'm not sure what it does so it  might be correct